Airport Ticket Booking/Airport Ticket Booking/Enums/UserType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace AirportTicketBooking.Enums
+{
+    public enum UserType
+    {
+        Passenger,
+        Mannager,
+        NotLoggedIn,
```

Comment:
In case not loggedIn users can use the system maybe it's better to call them `Guests`